### PR TITLE
feat: use acme.sh to provision ssl certificates

### PR DESCRIPTION
**Pull request by wenzowski**
untested; best guess at how to do this given it seems we can't use any of the supported k8s Let's Encrypt provisioners

### CODE REVIEW

**Comment by wenzowski** [comment]
related: should probably pull in [mkcert](https://github.com/FiloSottile/mkcert) so dev env has a [certificate for localhost](https://letsencrypt.org/docs/certificates-for-localhost/)

File: app/server/index.js [hunk 1]
```diff
@@ -294,11 +297,31 @@ app.prepare().then(() => {
     return handle(req, res);
   });
 
-  server.listen(port, (err) => {
-    if (err) {
-      throw err;
-    }
+  if (process.env.HOST.startsWith('https')) {
```
Comment (matthieu-foucault) [inline app/server/index.js:300]: `process.env.HOST` is `undefined` in our e2e test jobs. 
```suggestion
  if (process.env.HOST && process.env.HOST.startsWith('https')) {
```

Comment (wenzowski) [reply app/server/index.js:300]: goddamn prettier. I wrote `/^https/.test(process.env.HOST)`. fixing

**Comment by wenzowski** [comment]
@matthieu-foucault maybe we should consider turning off those two offending eslint rules? test does an implicit cast `String(arg)` of the passed argument where `startsWith` is only defined on the prototype of `String`. https://tc39.es/ecma262/#sec-regexp.prototype.test

That's `unicorn/prefer-starts-ends-with` & `@typescript-eslint/prefer-string-starts-ends-with`

The example I wrote will not match on null or undefined when cast as strings `String(null) => 'null'` & `String(undefined) => 'undefined'`. If truthiness is easier to read, can do the chaining you propose.

**Pull request merged**